vnext/ReactCommon/ReactCommon.vcxproj

```diff
@@ -189,6 +189,8 @@
     <!-- Forced include only necessary until we get https://github.com/facebook/react-native/pull/28023  back from upstream. -->
     <ClCompile Include="$(ReactNativeDir)\ReactCommon\turbomodule\core\TurboModuleBinding.cpp" ForcedIncludeFiles="stdexcept" />
     <ClCompile Include="$(ReactNativeDir)\ReactCommon\turbomodule\core\TurboModuleUtils.cpp" />
+    <CLCompile Include="$(ReactNativeDir)\ReactCommon\turbomodule\samples\NativeSampleTurboCxxModuleSpecJSI.cpp" />
```

Comment:
Can we avoid this change? We have a different implementation for TurboModules.

Reply:
I think I encountered a build issue, but I'll try.

Reply:
I was unable to make the solutions build without these sources.

I'll follow up in the next PR to exclude it form any projects it could be intrusive to.